dappnode/DNP_DAPPMANAGER
# Implement docker upgrade

Implement docker upgrade action from the UI:

- Script to check requirements:
  - Docker is in unattended upgrades
  - Docker host version
  - Docker latest version
  - Docker is installed through apt
- Script to upgrade docker
  - Check internet and dns resolution
  - Remove legacy docker packages
  - Get download urls depending on OS
  - Install new docker with apt
  - Add docker to unattended upgrades
  - Add docker alias
